application/lib/tutorial/six/tutorial_six.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:application/tutorial/six/module/type_with_virtual_keyboard.dart';
+import 'package:flutter/material.dart';
+import 'package:application/tutorial/six/module/chatView.dart';
+
+class TutorialSix extends StatelessWidget {
+  const TutorialSix({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    // scaffold page
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text("Tutorial 2"),
```

Comment:
```suggestion
        title: const Text("Tutorial 6"),
```

We'll probably want to do this numbering a bit more dynamically in the future, but not sure.